java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -48,4 +48,31 @@ public interface StringCommands {
      *     is set, return the old value as a <code>String</code>.
      */
     CompletableFuture<String> set(String key, String value, SetOptions options);
+
+    /**
+     * Increment the string representing a floating point number stored at <code>key</code> by <code>
```

Comment:
I think this is the comment for `incr`.  But `decr` should say "Decrements". 

Also, this command doesn't take an `amount`.  We should follow what's in the node side, since they have overloaded this function like we have. 

```
    /** Decrements the number stored at `key` by one. If `key` does not exist, it is set to 0 before performing the operation.
     * See https://redis.io/commands/decr/ for details.
     *
     * @param key - The key to decrement its value.
     * @returns the value of `key` after the decrement. An error is raised if `key` contains a value
     * of the wrong type or contains a string that can not be represented as integer.
     */
    public decr(key: string): Promise<number> {
        return this.createWritePromise(createDecr(key));
    }

    /** Decrements the number stored at `key` by `amount`. If `key` does not exist, it is set to 0 before performing the operation.
     * See https://redis.io/commands/decrby/ for details.
     *
     * @param key - The key to decrement its value.
     * @param amount - The amount to decrement.
     * @returns the value of `key` after the decrement. An error is raised if `key` contains a value
     * of the wrong type or contains a string that can not be represented as integer.
     */
    public decrBy(key: string, amount: number): Promise<number> {
        return this.createWritePromise(createDecrBy(key, amount));
    }
```

Reply:
I'll fix the documentation. I don't understand the rest of your comment decr just has 1 argument and that's the key what are you referring to? DecrBy has amount.

Reply:
The documentation used to mention "amount" for the overloaded method without "amount". All fixed now.  You're good.